using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -103,116 +119,150 @@ This API synchronizes the provided customer identifier type key and value with t
 ACPIdentity.syncIdentifier("idType", identifier: "idValue", authentication: ACPMobileVisitorAuthenticationState.unknown)
 ```
 
-## syncIdentifiers <a id="syncidentifiers"></a>
-
-Updates the provided customer IDs with the Adobe Experience Cloud ID Service.
+### syncIdentifiers<a id="syncIdentifiers-ios"></a>
 
-This API synchronizes the provided customer identifiers to the Adobe Experience Cloud ID Service. If a customer ID type matches an existing ID type, it is updated with the new ID value and [authentication state](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#authenticationstate). New customer IDs are added. These IDs are preserved between app upgrades, are saved and restored during the standard application backup process, and are removed at uninstall. If the current SDK privacy status is `optedout`, calling this method results in no operations being performed.
+The function of this `syncIdentifiers()`  API is same as the `syncIdentifier` API, which passes a list of identifiers as a dictionary each containing an identifier type as the key and an identifier as the value. In each of the identifiers pair, both identifier type and identifier should be non empty and non null values, otherwise they will be ignore. 
```

Comment:
- Please replace "this" with "the".
- Please put a period after "API" and delete the rest of the content.
- Make `syncIdentifiers` a hyperlink to the "syncIdentifers()" section title.